models/protocol_tests/generate.go

```diff
@@ -391,6 +458,8 @@ func GenerateAssertions(out interface{}, shape *api.Shape, prefix string) string
 			return fmt.Sprintf("assert.Equal(t, %#v, string(%s))\n", out, prefix)
 		case "integer", "long":
 			return fmt.Sprintf("assert.Equal(t, int64(%#v), *%s)\n", out, prefix)
+		case "jsonvalue":
```

Comment:
Is this condition still valid since the type will be a `aws.JSONValue` not a `map[string]interface{}`. Or is is it difference in this case?